### PR TITLE
[docs] misc updates

### PR DESCRIPTION
**Pull request by polarweasel**
This is a number of little fixes:
- Vale rule updates
- Meta docs (linting, mostly)
- Fixes to a couple docs for proper rendering

Netlify preview starts on the one that most needed an update, to get rid of a huge TOC right at the top...

@netlify /latest/develop/explore-sample-apps/

### CODE REVIEW

File: .github/vale-styles/Yugabyte/Spelling.yml [hunk 1]
```diff
@@ -17,8 +17,10 @@ extends: spelling
 message: 'Spelling: "%s"?'
 level: warning
 custom: false
+code: false
 filters:
   - '(?:\w*\.?)yugabyte\.com'
+  - '`.*?`'
```
Comment (polarweasel) [inline .github/vale-styles/Yugabyte/Spelling.yml:23]: This didn't work right, and I meant to remove it...
```suggestion
```

File: docs/content/latest/secure/column-level-encryption.md [hunk 2]
```diff
@@ -127,32 +119,23 @@ yugabyte=# select PGP_SYM_DECRYPT(account_number::bytea, 'AES_KEY') as AccountNu
...
 ```
 
-
 ## Asymmetric Encryption
 
-Asymmetric Encryption, also known as Public-Key cryptography can be used with YugabyteDB for enabling column level encryption. YugabyteDB can be configured with generated public/private keys or corporate gpg keys for encrypting column data.
-
+Asymmetric Encryption, also known as Public-Key cryptography can be used with YugabyteDB for enabling column-level encryption. YugabyteDB can be configured with generated public/private keys or corporate GPG keys for encrypting column data.
 
 The example below walks through configuring YugabyteDB cluster with a new set of keys (public and private)
```
Comment (ddhodge) [inline docs/content/latest/secure/column-level-encryption.md:126]: period

File: docs/content/latest/secure/column-level-encryption.md [hunk 3]
```diff
@@ -127,32 +119,23 @@ yugabyte=# select PGP_SYM_DECRYPT(account_number::bytea, 'AES_KEY') as AccountNu
...
 
+### Step 1. Generate an RSA key pair
 
-### Step 1. Generate RSA key pair
-
-* Start by generating new public and private RSA key pair using `gpg` key generator
-
-    ```
-    gpg --gen-key
+* Start by generating a new public and private RSA key pair using the `gpg` key generator
```
Comment (ddhodge) [inline docs/content/latest/secure/column-level-encryption.md:130]: period

File: docs/content/latest/secure/column-level-encryption.md [hunk 4]
```diff
@@ -1,4 +1,3 @@
-
 ---
 title: Column-Level Encryption
```
Comment (ddhodge) [inline docs/content/latest/secure/column-level-encryption.md:2]: headings and title sentence case?

File: docs/content/latest/secure/column-level-encryption.md [hunk 5]
```diff
@@ -161,81 +144,64 @@ The example below walks through configuring YugabyteDB cluster with a new set of
...
-    yugabyte=# CREATE EXTENSION IF NOT EXISTS pgcrypto;
-    CREATE EXTENSION
+    ```sql
+    create extension if not exists pgcrypto;
     ```
 
+### Step 3. Insert data using pgp_pub_encrypt
 
-### Step 3. Insert using `pgp_pub_encrypt`
+Create the `employees` table and insert data into the table using generated Public key for encrypting column data
```
Comment (ddhodge) [inline docs/content/latest/secure/column-level-encryption.md:180]: using the generated public

period

File: docs/content/latest/secure/column-level-encryption.md [hunk 6]
```diff
@@ -161,81 +144,64 @@ The example below walks through configuring YugabyteDB cluster with a new set of
...
-
-
-```
-yugabyte=# create table employees ( empno int, ename text, address text, salary int, account_number text );
-CREATE TABLE
+```sql
+create table employees ( empno int, ename text, address text, salary int, account_number text );
 ```
 
 In this example, account numbers of `employees` table will be encrypted using `pgp_pub_encrypt` function and the generated Public key,
```
Comment (ddhodge) [inline docs/content/latest/secure/column-level-encryption.md:186]: will be encrypted using -> are encrypted using the

comma -> period

File: docs/content/latest/secure/column-level-encryption.md [hunk 7]
```diff
@@ -127,32 +119,23 @@ yugabyte=# select PGP_SYM_DECRYPT(account_number::bytea, 'AES_KEY') as AccountNu
...
 
-* After going through configuration prompts, RSA key gets generated - 
+* After going through the configuration prompts, an RSA key gets generated:
 
-
-    ```
+    ```output
     public and secret key created and signed.
 
     pub   rsa2048 2020-11-09 [SC] [expires: 2022-11-09]
```
Comment (ddhodge) [inline docs/content/latest/secure/column-level-encryption.md:141]: curious spacing on uid line

File: docs/content/latest/secure/column-level-encryption.md [hunk 8]
```diff
@@ -161,81 +144,64 @@ The example below walks through configuring YugabyteDB cluster with a new set of
...
-
-* Next, export public and private key of newly generated RSA key
-
+* Next, export the public and private keys of the newly generated RSA key:
 
   * Private Key
 
-    ```
+    ```sh
       $ gpg --export-secret-keys \
```
Comment (ddhodge) [inline docs/content/latest/secure/column-level-encryption.md:152]: indentation

File: docs/content/latest/secure/column-level-encryption.md [hunk 9]
```diff
@@ -246,19 +212,21 @@ yugabyte=# select ename, account_number from employees limit 1;
 (1 row)
 ```
 
-### Step 5. Query using `pgp_pub_decrypt`
+### Step 5. Query using pgp_pub_decrypt
 
-Use `pgp_pub_decrypt` and private key for decrypting column data. In order to retrieve the encrypted column data, use `pgp_pub_decrypt` function to decrypt the data and wrap the pgp private key with dearmor function to convert the private key into PGP ASCII-armor format. The Decryption function needs to be used in both SELECT and WHERE clause depending on the query. 
+Use `pgp_pub_decrypt` and private key for decrypting column data. In order to retrieve the encrypted column data, use `pgp_pub_decrypt` function to decrypt the data and wrap the pgp private key with the `dearmor` function to convert the private key into PGP ASCII-armor format. The Decryption function needs to be used in both SELECT and WHERE clause depending on the query.
```
Comment (ddhodge) [inline docs/content/latest/secure/column-level-encryption.md:217]: Use `pgp_pub_decrypt` and the private key to decrypt column data. To retrieve the encrypted column data, use the `pgp_pub_decrypt` function to decrypt the data and wrap the PGP private key with the `dearmor` function to convert the private key into PGP ASCII-armor format. The Decryption function needs to be used in both SELECT and WHERE clauses, depending on the query.

File: docs/content/latest/secure/column-level-encryption.md [hunk 10]
```diff
@@ -246,19 +212,21 @@ yugabyte=# select ename, account_number from employees limit 1;
...
 
-To allow the decryption, the field name is also casted to the binary data type with the syntax: ` account_number:bytea`.
+To allow the decryption, the field name is also cast to the binary data type with the syntax: `account_number::bytea`.
```
Comment (ddhodge) [inline docs/content/latest/secure/column-level-encryption.md:219]: You also need to cast the field name to the binary data type by using the syntax `account_number::bytea`.

File: docs/content/latest/develop/explore-sample-apps.md [hunk 11]
```diff
@@ -13,6 +12,8 @@ menu:
     identifier: explore-sample-apps
     parent: develop
     weight: 581
+isTocNested: true
```
Comment (ddhodge) [inline docs/content/latest/develop/explore-sample-apps.md:15]: TOC is still pretty repetitive...

**Review by ddhodge — APPROVED** [review]
Misc minor comments

File: docs/content/latest/develop/explore-sample-apps.md (hunk 11 above)
Comment (polarweasel) [reply docs/content/latest/develop/explore-sample-apps.md:15]: At least it's on the side though, I guess...

**Pull request merged**